data_steward/cdr_cleaner/cleaning_rules/drug_refills_days_supply.py

```diff
@@ -1,68 +1,134 @@
+
+Original Issues: DC-403, DC-815
 """
+# Python imports
 import logging
 
 # Project imports
-import common
-from constants import bq_utils as bq_consts
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
 from constants.cdr_cleaner import clean_cdr as cdr_consts
+from common import DRUG_EXPOSURE, JINJA_ENV
+from utils import pipeline_logging
 
 LOGGER = logging.getLogger(__name__)
 
 MAX_DAYS_SUPPLY = 180
 MAX_REFILLS = 10
-drug_exposure = common.DRUG_EXPOSURE
-
-MAX_DAYS_SUPPLY_AND_REFILLS_QUERY = (
-    'SELECT * '
-    'FROM `{project_id}.{dataset_id}.drug_exposure` '
-    'WHERE ((days_supply <= {MAX_DAYS_SUPPLY} or days_supply is null) '
-    '       AND (REFILLS <= {MAX_REFILLS} or REFILLS IS NULL))')
-
-
-def get_days_supply_refills_queries(project_id,
-                                    dataset_id,
-                                    sandbox_dataset_id=None):
-    """
-    This function gets the queries required to remove table records which are prior
-    to the person's birth date or 150 years past the birth date from a dataset
-
-    :param project_id: Project name
-    :param dataset_id: Name of the dataset where a rule should be applied
-    :param sandbox_dataset_id: Identifies the sandbox dataset to store rows 
-    #TODO use sandbox_dataset_id for CR
-    :return: a list of queries.
-    """
-    queries = []
-    query = dict()
-    query[cdr_consts.QUERY] = MAX_DAYS_SUPPLY_AND_REFILLS_QUERY.format(
-        project_id=project_id,
-        dataset_id=dataset_id,
-        MAX_DAYS_SUPPLY=MAX_DAYS_SUPPLY,
-        MAX_REFILLS=MAX_REFILLS)
-    query[cdr_consts.DESTINATION_TABLE] = drug_exposure
-    query[cdr_consts.DISPOSITION] = bq_consts.WRITE_TRUNCATE
-    query[cdr_consts.DESTINATION_DATASET] = dataset_id
-    queries.append(query)
-    return queries
+
+SANDBOX_QUERY = JINJA_ENV.from_string("""
+CREATE OR REPLACE TABLE `{{project_id}}.{{sandbox_dataset_id}}.{{sandbox_table_id}}` AS (
+    SELECT * FROM `{{project_id}}.{{dataset_id}}.drug_exposure`
+    WHERE {{max_days_supply}} < days_supply
+    OR {{max_refills}} < refills
+)
+""")
+
+DELETE_QUERY = JINJA_ENV.from_string("""
+DELETE FROM `{{project_id}}.{{dataset_id}}.drug_exposure` 
+WHERE drug_exposure_id IN (
+    SELECT drug_exposure_id 
+    FROM `{{project_id}}.{{sandbox_dataset_id}}.{{sandbox_table_id}}`
+)
+""")
+
+
+class DrugRefillsDaysSupply(BaseCleaningRule):
+
+    def __init__(self,
+                 project_id,
+                 dataset_id,
+                 sandbox_dataset_id,
+                 table_namer=None):
+        """
+        Initialize the class with proper information.
+        """
+        desc = ('Ensure drug refills < 10 and days_supply < 180.')
+        super().__init__(issue_numbers=['DC403', 'DC815'],
+                         description=desc,
+                         affected_datasets=[cdr_consts.COMBINED],
+                         affected_tables=DRUG_EXPOSURE,
```

Comment:
This needs to be a list.
```suggestion
                         affected_tables=[DRUG_EXPOSURE],
```